============================= test session starts ==============================
platform linux -- Python 3.10.12, pytest-8.3.3, pluggy-1.5.0
rootdir: /home/james/projects/cs6223/our_proptest_data
plugins: hypothesis-6.112.2, timeout-2.3.1
collected 20 items

code_only/mutants_2/statistics.mean/mutant_1.py FFFFF                    [ 25%]
code_only/mutants_2/statistics.mean/mutant_2.py F.FFF                    [ 50%]
code_only/mutants_2/statistics.mean/mutant_3.py FFFFF                    [ 75%]
code_only/mutants_2/statistics.mean/mutant_4.py F..FF                    [100%]

=================================== FAILURES ===================================
/home/james/projects/cs6223/our_proptest_data/code_only/mutants_2/statistics.mean/mutant_1.py:10: assert 1.0 == 0.0
/home/james/projects/cs6223/our_proptest_data/code_only/mutants_2/statistics.mean/mutant_1.py:17: assert 2.0 == 1.0
/home/james/projects/cs6223/our_proptest_data/code_only/mutants_2/statistics.mean/mutant_1.py:24: assert 0.0 == 1.0
/home/james/projects/cs6223/our_proptest_data/code_only/mutants_2/statistics.mean/mutant_1.py:31: assert 2.0 == 1.0
/home/james/projects/cs6223/our_proptest_data/code_only/mutants_2/statistics.mean/mutant_1.py:38: assert 10.0 == 0.0
/home/james/projects/cs6223/our_proptest_data/code_only/mutants_2/statistics.mean/mutant_2.py:6: hypothesis.errors.FailedHealthCheck: Tests run under @given should return None, but test_violation_of_statistics_mean_1 returned 0.0 instead.
/home/james/projects/cs6223/our_proptest_data/code_only/mutants_2/statistics.mean/mutant_2.py:18: hypothesis.errors.FailedHealthCheck: Tests run under @given should return None, but test_violation_of_statistics_mean_3 returned 'mean' instead.
/home/james/projects/cs6223/our_proptest_data/code_only/mutants_2/statistics.mean/mutant_2.py:24: hypothesis.errors.FailedHealthCheck: Tests run under @given should return None, but test_violation_of_statistics_mean_4 returned -1.0 instead.
/home/james/projects/cs6223/our_proptest_data/code_only/mutants_2/statistics.mean/mutant_2.py:30: hypothesis.errors.FailedHealthCheck: Tests run under @given should return None, but test_violation_of_statistics_mean_5 returned 10000000000.0 instead.
/home/james/projects/cs6223/our_proptest_data/code_only/mutants_2/statistics.mean/mutant_3.py:8: assert (10.0 >= 0.0 and 10.0 <= 0.0)
/home/james/projects/cs6223/our_proptest_data/code_only/mutants_2/statistics.mean/mutant_3.py:13: assert (-10.0 >= 0.0)
/home/james/projects/cs6223/our_proptest_data/code_only/mutants_2/statistics.mean/mutant_3.py:18: assert (2.0 >= 1.0 and 2.0 <= 1.0)
/home/james/projects/cs6223/our_proptest_data/code_only/mutants_2/statistics.mean/mutant_3.py:23: assert (1.0 >= 0.0 and 1.0 <= 0.0)
/home/james/projects/cs6223/our_proptest_data/code_only/mutants_2/statistics.mean/mutant_3.py:28: assert (-1.0 >= 1.0)
/home/james/projects/cs6223/our_proptest_data/code_only/mutants_2/statistics.mean/mutant_4.py:13: assert 1.0 == 0.0
/home/james/projects/cs6223/our_proptest_data/code_only/mutants_2/statistics.mean/mutant_4.py:43: assert 1.5 == 0.5
/home/james/projects/cs6223/our_proptest_data/code_only/mutants_2/statistics.mean/mutant_4.py:53: assert 0.25 == 0.5
=========================== short test summary info ============================
FAILED code_only/mutants_2/statistics.mean/mutant_1.py::test_violation_of_statistics_mean_1
FAILED code_only/mutants_2/statistics.mean/mutant_1.py::test_violation_of_statistics_mean_2
FAILED code_only/mutants_2/statistics.mean/mutant_1.py::test_violation_of_statistics_mean_3
FAILED code_only/mutants_2/statistics.mean/mutant_1.py::test_violation_of_statistics_mean_4
FAILED code_only/mutants_2/statistics.mean/mutant_1.py::test_violation_of_statistics_mean_5
FAILED code_only/mutants_2/statistics.mean/mutant_2.py::test_violation_of_statistics_mean_1
FAILED code_only/mutants_2/statistics.mean/mutant_2.py::test_violation_of_statistics_mean_3
FAILED code_only/mutants_2/statistics.mean/mutant_2.py::test_violation_of_statistics_mean_4
FAILED code_only/mutants_2/statistics.mean/mutant_2.py::test_violation_of_statistics_mean_5
FAILED code_only/mutants_2/statistics.mean/mutant_3.py::test_violation_of_statistics_mean_1
FAILED code_only/mutants_2/statistics.mean/mutant_3.py::test_violation_of_statistics_mean_2
FAILED code_only/mutants_2/statistics.mean/mutant_3.py::test_violation_of_statistics_mean_3
FAILED code_only/mutants_2/statistics.mean/mutant_3.py::test_violation_of_statistics_mean_4
FAILED code_only/mutants_2/statistics.mean/mutant_3.py::test_violation_of_statistics_mean_5
FAILED code_only/mutants_2/statistics.mean/mutant_4.py::test_violation_of_statistics_mean_1
FAILED code_only/mutants_2/statistics.mean/mutant_4.py::test_violation_of_statistics_mean_4
FAILED code_only/mutants_2/statistics.mean/mutant_4.py::test_violation_of_statistics_mean_5
========================= 17 failed, 3 passed in 2.09s =========================
